xyzspaces/apis.py

```diff
@@ -513,12 +513,17 @@ def delete_space(self, space_id: str) -> str:
     # Read Features
 
     def get_space_features(
-        self, space_id: str, feature_ids: List[str]
+        self,
+        space_id: str,
+        feature_ids: List[str],
+        force_2d: Optional[bool] = None,
     ) -> dict:
         """Get features by ID.
 
         :param space_id: The desired space ID.
         :param feature_ids: A list of feature_ids.
+        :param force_2d: If set to true the features in the response
```

Comment:
It should be `If set to True` Also add by default all x,y,z coordinates will be returned. change this in all places.